tooling/cli.rs/src/info.rs

```diff
@@ -306,63 +244,66 @@ fn webview2_version() -> crate::Result<Option<String>> {
 }
 
 #[cfg(windows)]
-fn run_vs_setup_instance() -> std::io::Result<std::process::Output> {
-  Command::new("powershell")
-    .args(&["-NoProfile", "-Command"])
-    .arg("Get-VSSetupInstance")
-    .output()
+#[derive(Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+struct VsInstanceInfo {
+  display_name: String,
 }
 
 #[cfg(windows)]
-fn build_tools_version() -> crate::Result<Option<Vec<String>>> {
-  let mut output = run_vs_setup_instance();
-  if output.is_err() {
-    Command::new("powershell")
-      .args(&["-NoProfile", "-Command"])
-      .arg("Install-Module VSSetup -Scope CurrentUser")
-      .output()?;
-    output = run_vs_setup_instance();
-  }
-  let output = output?;
-  let versions = if output.status.success() {
-    let stdout = String::from_utf8_lossy(&output.stdout);
-    let mut versions = Vec::new();
+const VSWHERE: &[u8] = include_bytes!("../vswhere.exe");
```

Comment:
how often do you think we'll update this binary? I think we won't need to update it ever but if we do I think we could move it to binary-releases (just like we did with rustup).

Reply:
maybe once every year or two, it is very unlikely to update this binary, it is also hosted in github releases here https://github.com/microsoft/vswhere/releases so no need to host it ourselves.